service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/AddressMapper.java

```diff
@@ -21,11 +25,17 @@ public Address mapAddress(AD itkAddress) {
         Address address = new Address();
 
         Arrays.stream(itkAddress.getStreetAddressLineArray())
-                .map(nodeUtil::getNodeValueString)
-                .forEach(address::addLine);
+            .map(nodeUtil::getNodeValueString)
+            .forEach(address::addLine);
 
         if (itkAddress.isSetUse()) {
-            address.setUse(AddressUse.fromCode(getAddressUseString(itkAddress.getUse().get(0).toString())));
+            String addressTypeString = getAddressTypeString(itkAddress.getUse().get(0).toString());
```

Comment:
Because we set type only in one case when use equals 'PHYS', I think something like this if statement could be enough:
```
if (itkAddress.getUse().get(0).toString().equals("PHYS")) {
    address.setType(AddressType.fromCode("physical"));
}
```